fetch/main.js

```diff
@@ -0,0 +1,21 @@
+async function logData() {
+  try {
+    const response = await fetch('https://jsonplaceholder.typicode.com/users');
+    const jsonData = await response.json();
```

Comment:
The response will have a field named `ok` that will be false if the server returned an error. To do proper error handling with fetch, you always need to check `response.ok` before you get the JSON from the response.

The best way to do this is to add this line of code right after you call fetch:
```if (!response.ok) throw new Error(`fetch Error ${response.status}`);```